.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.nr linguist-language=rust
```

Comment:
At this moment github classifies this project as Roff language. Let's make it Rust like in [Noir](https://github.com/noir-lang/noir/blob/master/.gitattributes)